### PR TITLE
Create UI for Sign In Panel

### PR DESCRIPTION
**Pull request by DongTrieu1998**

### CODE REVIEW

File: fudy/FudyComponents/ClickableText.qml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+		MouseArea {
+			anchors.fill: parent
+			hoverEnabled: true
+
+			onContainsMouseChanged: function() {
+
+			}
+
+			onClicked: function() {
+
```
Comment (ghost) [inline fudy/FudyComponents/ClickableText.qml:31]: empty?

File: fudy/NoteScreen/Component/FudyButton.qml [hunk 2]
```diff
@@ -1,31 +1,51 @@
 import QtQuick 2.12
 import QtQuick.Controls 2.12
```
Comment (ghost) [inline fudy/NoteScreen/Component/FudyButton.qml:2]: remove version

File: fudy/SignInScreen/InputBox.qml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+			id: title
+			text: root.title
+			font: FudyFont.heading2
+		}
+
+		Rectangle {
+			width: root.width
+			height: 48
+			radius: 12
+			color: "#F7FBFF"
```
Comment (ghost) [inline fudy/SignInScreen/InputBox.qml:31]: please change

File: fudy/SignInScreen/SignInPanel.qml [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+
+		ClickableText {
+			id: clickableText
+
+			Layout.alignment: Qt.AlignRight
+			text: qsTr("Forgot Password?")
+		}
+
+		FudyButton {
+
```
Comment (ghost) [inline fudy/SignInScreen/SignInPanel.qml]: -1

File: fudy/SignInScreen/SignInPanel.qml [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+			Layout.fillWidth: true
+			Layout.preferredHeight: 52
+
+			textColor: "#FFFFFF"
+			textPressColor: "#FFFFFF"
+			backgroundColor: "#162D3A"
+			backgroundPressColor: "#162D3A"
```
Comment (ghost) [inline fudy/SignInScreen/SignInPanel.qml:76]: ?

File: fudy/SignInScreen/SignInPanel.qml [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+
+	RowLayout {
+
+		Layout.alignment: Qt.AlignCenter
+		spacing: 16
+
+		SplitterItem {
+			width: 164
+			Layout.alignment: Qt.AlignVCenter
+			color: "#CFDFE2"
```
Comment (ghost) [inline fudy/SignInScreen/SignInPanel.qml:114]: ?

File: fudy/SignInScreen/SignInPanel.qml [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+		}
+	}
+
+	FudyButton {
+
```
Comment (ghost) [inline fudy/SignInScreen/SignInPanel.qml]: -1

File: fudy/SignInScreen/SignInPanel.qml [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+		Text {
+			Layout.alignment: Qt.AlignVCenter
+			text: qsTr("Or")
+			font: FudyFont.text1
+		}
+
+		SplitterItem {
+			width: 164
+			Layout.alignment: Qt.AlignVCenter
+			color: "#CFDFE2"
```
Comment (ghost) [inline fudy/SignInScreen/SignInPanel.qml:126]: ?

File: fudy/SignInScreen/SignInPanel.qml [hunk 9]
```diff
@@ -0,0 +1,145 @@
...
+
+	FudyButton {
+
+		Layout.fillWidth: true
+		Layout.preferredHeight: 52
+
+		textColor: "#313957"
+		textPressColor: "#313957"
+		backgroundColor: "#F3F9FA"
+		backgroundPressColor: "#F3F9FA"
```
Comment (ghost) [inline fudy/SignInScreen/SignInPanel.qml:137]: ?

File: fudy/SignInScreen/SignInScreen.qml [hunk 10]
```diff
@@ -0,0 +1,59 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Window 2.15
+import QtQuick.Layouts
```
Comment (ghost) [inline fudy/SignInScreen/SignInScreen.qml:4]: version

File: fudy/FudyComponents/ClickableText.qml (hunk 1 above)
Comment (DongTrieu1998) [reply fudy/FudyComponents/ClickableText.qml:31]: This was updated in the next commit. You could check it.

File: fudy/NoteScreen/Component/FudyButton.qml (hunk 2 above)
Comment (DongTrieu1998) [reply fudy/NoteScreen/Component/FudyButton.qml:2]: I will fix it in a commit for refactoring.

File: fudy/SignInScreen/InputBox.qml (hunk 3 above)
Comment (DongTrieu1998) [reply fudy/SignInScreen/InputBox.qml:31]: This was updated in the next commit.

File: fudy/SignInScreen/SignInPanel.qml (hunk 4 above)
Comment (DongTrieu1998) [reply fudy/SignInScreen/SignInPanel.qml]: I don't understand what you mean.

File: fudy/SignInScreen/SignInPanel.qml (hunk 5 above)
Comment (DongTrieu1998) [reply fudy/SignInScreen/SignInPanel.qml:76]: This was updated in the next commit.

File: fudy/SignInScreen/SignInPanel.qml (hunk 6 above)
Comment (DongTrieu1998) [reply fudy/SignInScreen/SignInPanel.qml:114]: This was updated in the next commit.

File: fudy/SignInScreen/SignInPanel.qml (hunk 7 above)
Comment (DongTrieu1998) [reply fudy/SignInScreen/SignInPanel.qml]: Do you mean removing the blank?

File: fudy/SignInScreen/SignInPanel.qml (hunk 8 above)
Comment (DongTrieu1998) [reply fudy/SignInScreen/SignInPanel.qml:126]: You can check the comment above.

File: fudy/SignInScreen/SignInPanel.qml (hunk 9 above)
Comment (DongTrieu1998) [reply fudy/SignInScreen/SignInPanel.qml:137]: You can check the comment above.

File: fudy/SignInScreen/SignInScreen.qml (hunk 10 above)
Comment (DongTrieu1998) [reply fudy/SignInScreen/SignInScreen.qml:4]: You can check the comment above.

File: fudy/SignInScreen/SignInPanel.qml (hunk 4 above)
Comment (DongTrieu1998) [reply fudy/SignInScreen/SignInPanel.qml]: The end line was removed.

File: fudy/SignInScreen/SignInPanel.qml (hunk 7 above)
Comment (DongTrieu1998) [reply fudy/SignInScreen/SignInPanel.qml]: The end line was removed.

**Review by ghost — APPROVED** [review]

**Pull request merged**